conf/messages.en

```diff
@@ -297,11 +297,9 @@ results.legend.high = High
 results.legend.low = Low
 results.legend.body = Gray neighborhoods lack sufficient data to assess sidewalk problems. Click on any neighborhood to start contributing. You can explore the data yourself using <a href="/api" target="_blank">our APIs</a>!
 
-dashboard.no.org = No Organization
-dashboard.edit.org = Edit
-dashboard.edit.info = If you don''t see your organization, please email <a id="email-link" href="mailto:sidewalk@cs.uw.edu" target="_blank">sidewalk@cs.uw.edu</a> and include your organization name and description.
-dashboard.orgs.unavailable = There are no available organizations!
-dashboard.leave.org = Leave Organization
+dashboard.leave.team=Leave team:
```

Comment:
Let's actually pass the team name in as a parameter here, to account for situations in the future where the sentence structure might be different in different languages! I think it would look like this?
```
dashboard.leave.team = Leave team: {0}
```

Reply:
@misaugstad I'm actually doing it within the scala code. I'm displaying the leave team: (name of organization) in that format. I tried to pass in the parameter but couldn't get it to work which is why I did it that why and that's why its not in the .language files. I'll give it another shot this round though, I'll update you on this!

Reply:
The issue is if we had a language with a different sentence structure, such that the team name didn't go at the end! With the way you've done it, it's required to go at the end. I don't expect it to be a problem here, but it's just good practice to insert it this way. Hoping it goes more smoothly this time!

Reply:
OHHH that makes sense! Sorry didn't full understand your comment, thanks for insight :)